cmd/podmanV2/common/default.go

```diff
@@ -12,6 +12,19 @@ import (
 	"github.com/opencontainers/selinux/go-selinux"
 )
 
+var (
+	// DefaultHealthCheckInterval default value
+	DefaultHealthCheckInterval = "30s"
```

Comment:
These should probably go into containers.conf eventually, but this is fine for now.